packages/controllers/src/permissions/PermissionController.ts

```diff
@@ -325,7 +377,7 @@ export type ExtractPermission<
  * caveat specifications available to the controller.
  */
 export type PermissionControllerOptions<
-  ControllerPermissionSpecification extends ValidPermissionSpecification<PermissionSpecificationConstraint>,
```

Comment:
Why was `ValidPermissionSpecification` removed here? Not disagreeing, just the reason wasn't clear to me.

Reply:
Ah, `ValidPermissionSpecification<PermissionSpecificationConstraint>` evaluates to `never` with the new types when used as the right-hand operand of an `extends` expression. The other uses of `ValidPermissionSpecification` still work, and it didn't really add anything here, so removing it seemed like best move.